LDAR_Sim/testing/unit_testing/test_virtual_world/test_nonfugitive_emissions/nonfugitive_emission_testing_fixtures.py

```diff
@@ -0,0 +1,113 @@
+from datetime import date
+from typing import Tuple
+import pytest
+
+from src.virtual_world.nonfugitive_emissions import NonRepairableEmission
+
+
+@pytest.fixture(name="mock_simple_nonfugitive_emission_for_activate_testing_1")
+def mock_simple_nonfugitive_emission_for_activate_testing_1_fix() -> NonRepairableEmission:
+    return NonRepairableEmission(1, 1, date(*[2018, 1, 1]), date(*[2017, 1, 1]), False, {}, 365)
```

Comment:
This doesn't need to be part of this PR but that second date seems not to be being used, we should remove it